argilla-server/src/argilla_server/search_engine/commons.py

```diff
@@ -662,7 +662,7 @@ def _build_text_query(dataset: Dataset, text: Optional[Union[TextQuery, str]] =
             if field is None:
                 raise Exception(f"Field {text.field} not found in dataset {dataset.id}")
 
-            if field.is_chat:
+            if field.is_chat or field.is_custom:
```

Comment:
@frascuchon should this be included in the release? perhaps we missed this initially?